### ISSUE TITLE
A question about FID in places dataset

### ISSUE THREAD

**Issue by shallweiwei** [fenglinglwb/MAT#63]
Thanks for your excellent work!
I am a beginner in this field.
I use your pretrained model to finetune.And it gives me FID as first, it's 3.03.And the paper is 1.96.
So as the masks are different every time,maybe we should use the same masks as we compare methods?

**Comment by fenglinglwb** [comment]
As illustrated in the paper, we should use the same masks for evaluating all models for a fair comparison. We have provided the testing masks. And you could download them and specify the folder path to dataset_val.py.

**Comment by shallweiwei** [comment]
Thank you for your quick answer.
And are those masks made by the same code in the project?Because there is a big gap between 3.03 and 1.96

**Comment by fenglinglwb** [comment]
We use the same code to create masks. FID is largely affected by the number of images. Do you evaluate all 36500 validation images?
You could test our models and masks. The FID should be 1.96 with minor influctuation.

**Comment by shallweiwei** [comment]
> Do you evaluate all 36500 validation images?

Yes as I am using your pretrained model to train,so it gives me FID at first.

> You could test our models and masks.

So it means I need to use the generate_image.py file first ,then I should use cal_fid_pids_uids.py as it's not easy to use metrics like in training.There are too many kwargs

**Comment by fenglinglwb** [comment]
You could directly specify the testing masks during training here:
https://github.com/fenglinglwb/MAT/blob/d273d891ecdad2e1df106516423a75bc45b2d800/datasets/dataset_512_val.py#L192

**Comment by shallweiwei** [comment]
Thanks for your reply,I'll try it. And hoping you can make more excellent works.
